clients/go/cmd/zbctl/main_test.go

```diff
@@ -126,6 +126,20 @@ var tests = []testCase{
 		goldenFile: "testdata/create_instance.golden",
 		jsonOutput: true,
 	},
+	{
+		name:       "evaluate decision with decision id",
+		setupCmds:  [][]string{strings.Fields("--insecure deploy testdata/drg-force-user.dmn")},
+		cmd:        strings.Fields("--insecure create decision jedi_or_sith --variables testdata/decision_variables.json"),
+		goldenFile: "testdata/evaluate_decision.golden",
+		jsonOutput: true,
+	},
+	{
+		name:       "evaluate decision with decision key",
+		setupCmds:  [][]string{strings.Fields("--insecure deploy testdata/drg-force-user.dmn")},
+		cmd:        strings.Fields("--insecure create decision 2251799813685270 --variables testdata/decision_variables.json"),
```

Comment:
@remcowesterhoud let me know if you would prefer one of the test cases to pass variables explicitly, instead of as a file.

Reply:
I think it would be nice to do this. It's a slightly different behaviour I suppose.

Reply:
It would be nice to add this yes, it's a slightly different behaviour so let's make sure we cover it.